app/views/instruments/show.html.erb

```diff
@@ -1 +1,3 @@
 <h2><%= @instrument.name %></h2>
+
+<p><%= link_to "Click here to book this instrument", bookings_path %></p>
```

Comment:
Do we have to put something like:     bookings_path(instrument) ?
I really dont know...
That link will take the user to the booking, but how can we specify that the user is booking that specific instrument?

Reply:
```suggestion
<p><%= link_to "Click here to book this instrument", bookings_path(`@instrument`) %></p>
```

Yeah makes a lot of sense. Because the booking is for a specific instrument, so the id is included in the path.